app/views/partials/lib/consumers/reply_created/activity_feeds.liquid

```diff
@@ -0,0 +1,20 @@
+{% comment %}
+  actor:private actor:public actor:audience
+  object:private
+{% endcomment %}
+
+{% liquid
+  log event, type: 'event'
+
+  function audience = 'lib/queries/activities/users/audience', id: event.actor.id
+  render 'lib/commands/activities/publish', uuid: event.uuid, type: "user-private", ids: audience
```

Comment:
I would use `function` instead of `render`. Lets use `render` for rendering views only.